sentry-spring/api/sentry-spring.api

```diff
@@ -54,6 +54,12 @@ public class io/sentry/spring/SentrySpringServletContainerInitializer : javax/se
 	public fun onStartup (Ljava/util/Set;Ljavax/servlet/ServletContext;)V
 }
 
+public class io/sentry/spring/SentryUserFilter : javax/servlet/Filter {
```

Comment:
Looks like we need to add these to the docs and a note on a migration plan for the next version (should be 5.0 I believe)?